source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -170,7 +185,12 @@ def create_pb_request(requested_type, request, dep_mode, arr_mode):
     sn_params.origin_mode = dep_mode
     sn_params.destination_mode = arr_mode
 
-    req.journeys.max_duration = request["max_duration"]
+    # If we only want direct_paths, max_duration(time to pass in pt) is zero
+    if dp_type == "only":
+        req.journeys.max_duration = 0
```

Comment:
doing a `DirectPathRequest` would probably be more efficient, and I don't think it will change the response

Reply:
Tried it but the direct paths are never returned. :(